src/bot/commands/approve.ts

```diff
@@ -115,121 +178,124 @@ export default class ApproveCommand extends Command {
             disapprovals: true
           }
         },
-        report: {
-          include: {
-            corrections: {
-              orderBy: {
-                created_at: 'desc'
-              },
-              where: {
-                merged: false,
-                refused: false,
-                stale: false
-              }
-            },
-            approvals: true,
-            disapprovals: true
-          }
-        },
+
         approvals: true,
         disapprovals: true,
         votes: true
       }
-    })) as Proposals | null;
+    })) as Exclude<Proposals, ReportExtended> | null;
 
     if (!proposal) {
-      return interaction.reply(interactionProblem(`Le message est invalide.`));
+      await interaction.reply(interactionProblem(`Le message est invalide.`));
+      return null;
     }
 
     const isSuggestion = proposal.type === ProposalType.SUGGESTION;
-    const isReport = proposal.type === ProposalType.REPORT;
 
     const embed = message.embeds[0]?.toJSON();
     if (!embed) {
       await prisma.proposal.delete({
         where: {
-          id: proposal.id
+          id: proposal.id!
```

Comment:
id n'est pas optionel (coucou le typage forcé qui est faux)
```suggestion
          id: proposal.id
```